create_box/main.py

```diff
@@ -1,9 +1,17 @@
 """This is the entry point of the program."""
 
-
 def create_box(height, width, character):
-    pass
+	box = ''
+	line = width * character + '\n'
+	counter = 0
+	while counter < height:
```

Comment:
The more pythonic way of accomplishing this would be something like `for h in range(height):`. This will loop through exactly `height` times without your code having to maintain a counter.